pyproject.toml

```diff
@@ -0,0 +1,30 @@
+[build-system]
+requires = ["setuptools>=61.0", "easycheck"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "makepackage"
+version = "0.2.0"
+authors = [{name="Nyggus", email="nyggus@gmail.com"},]
+description = "Creating a structure of a simple Python package"
+readme = "README.md"
+license = {file = "LICENSE"}
+requires-python = ">=3.6"
+dependencies = ["easycheck"]
+classifiers=[
+        "Programming Language :: Python :: 3",
+        "License :: OSI Approved :: MIT License",
+        "Operating System :: OS Independent",
+]
+
+[project.urls]
+"Homepage" = "https://github.com/nyggus/makepackage"
+
+[tool.setuptools]
+packages = ["makepackage"]
+
+[project.scripts]
+makepackage = "makepackage.__main__:main"
+
+[project.optional-dependencies]
+cli = ["wheel", "black", "pytest", "mypy"]
```

Comment:
Why `cli`? In setup.py, this was `dev`, and it makes sense, since these packages are used in developing `makepackage` but should not be installed when one install `makepackage` from PyPi. Hence, I think this should be `dev`.